crates/swc_ecma_minifier/tests/terser/compress/switch/gut_entire_switch/config.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Did you copied this from terser test suite?
Otherwise it should be moved to `tests/fixture` instead of `tests/terser`

Reply:
Yes